### ISSUE TITLE
Adding delay for Native dsd playing (switching between PCM and DSD)

### ISSUE THREAD

**Issue by norrest** [lintweaker/mpd-dsd-019#19]
Hi! On squeezelite was some code for delay 
I think its resolve problem with pop/ckick`s dsd! 
Can u do some option for this purpose?  

> ```
>  "  \t\t\t phase_response = 0-100 (0 = minimum / 50 = linear / 100 = maximum)\n"
> ```
> 
>  #endif
>  #if DSD
> -        "  -D [delay]\t\tOutput device supports DSD over PCM (DoP), delay = optional delay switching between PCM and DoP in ms\n" 
>   +#if ALSA
> -        "  -D [delay][:format]\tOutput device supports DSD, delay = optional delay switching between PCM and DSD in ms\n"
> -        "  \t\t\t format = dop (default if not specified), u8, u16le, u16be, u32le or u32be.\n"
>   +#else
> -        "  -D [delay]\t\tOutput device supports DSD over PCM (DoP), delay = optional delay switching between PCM and DoP in ms\n"
> 
> http://www.daphile.com/gpl-src/00-Daphile-patches/squeezelite/native-dsd-support.patch

[dsd.txt](https://github.com/lintweaker/mpd-dsd-019/files/169524/dsd.txt)

**Comment by norrest** [comment]
I I have a conflict with the developer of the MPD, which does not understand .....
As a result, he deleted my account and closed my  all task.

https://bugs.musicpd.org/view.php?id=4510
https://bugs.musicpd.org/view.php?id=4519

In the description it was the solution.

**Comment by norrest** [comment]
And yet, I noticed at the following situation.
The DoP mode - DAC operates worst. When you switch music tracks or rewind music - DAC click with the relay. The sound is also heard! If in native mode - when rewinding the relay is no longer clicks. But in the sound are present at the clicks, but not so much. But alas, in Windows systems clicks are not present  - if used dsd native. I wonder why? 
@lintweaker Maybe your patch, has some that feature? I use the latets - Alsa driver, mpd, kernel 4.4.8. I tried to change the various settings of the buffer - does not help.
Maybe there are nuances to the settings
period, periodsize?
You use these settings in your patches!
I would be very grateful for the help!
